tooling/v8-snapshot/cache/darwin/snapshot-meta.json

```diff
@@ -960,7 +960,7 @@
     "./packages/socket/node_modules/socket.io/node_modules/ws/lib/websocket.js",
     "./packages/ts/register.js",
     "./packages/types/index.js",
-    "./tooling/v8-snapshot/dist/setup/v8-snapshot-entry-cy-in-cy.js"
+    "./tooling/v8-snapshot/dist/setup/v8-snapshot-entry.js"
```

Comment:
Let's revert these changes. This is only valid really for the binary build.

Reply:
```suggestion
    "./tooling/v8-snapshot/dist/setup/v8-snapshot-entry-cy-in-cy.js"
```